frontend/src/query_builder/ExtendedOptions.jsx

```diff
@@ -182,23 +182,23 @@ export default class ExtendedOptions extends Component {
                     {this.renderSort()}
 
                     {_.contains(tableMetadata.db.features, "expressions") ?
-                        <Expressions
-                            expressions={query.query.expressions}
-                            tableMetadata={tableMetadata}
-                            onAddExpression={() => this.setState({isOpen: false, editExpression: true})}
-                            onEditExpression={(name) => {
-                                this.setState({isOpen: false, editExpression: name});
-                                MetabaseAnalytics.trackEvent("QueryBuilder", "Show Edit Custom Field");
-                            }}
-                        />
-                    : null}
-
-                    { features.limit &&
-                        <div>
-                            <div className="mb1 h6 text-uppercase text-grey-3 text-bold">Row limit</div>
-                            <LimitWidget limit={query.query.limit} onChange={this.setLimit} />
-                        </div>
-                    }
+                     <Expressions
+                         expressions={query.query.expressions}
+                         tableMetadata={tableMetadata}
+                         onAddExpression={() => this.setState({isOpen: false, editExpression: true})}
+                         onEditExpression={(name) => {
+                                 this.setState({isOpen: false, editExpression: name});
+                                 MetabaseAnalytics.trackEvent("QueryBuilder", "Show Edit Custom Field");
+                             }}
+                     />
+                     : null}
+
+                     { features.limit &&
+                       <div>
+                           <div className="mb1 h6 text-uppercase text-grey-3 text-bold">Row limit</div>
+                           <LimitWidget limit={query.query.limit} onChange={this.setLimit} />
+                       </div>
+                     }
```

Comment:
whoops I guess I must have forgot to put

```emacs-lisp
(setq follow-clojure-norms-for-indenting-javascript nil)
```

in my `init.el`.

On a serious note this was accidental and has nothing to do clojure formatting